### DISCUSSION TITLE
How to build mitmdump with different Python version?

### DISCUSSION THREAD

**Discussion by Shnitzelil** [mitmproxy/mitmproxy#6683] [Q&A]
In general how to build mitmdump with my own AddOns and shipit as single binary?

**Comment by Prinzhorn** [comment]
All relevant things should be in:

https://github.com/mitmproxy/mitmproxy/blob/240a286b2a3cedd0193b660e5217adfa46bb82b3/release/build.py
https://github.com/mitmproxy/mitmproxy/blob/240a286b2a3cedd0193b660e5217adfa46bb82b3/.github/workflows/main.yml
https://github.com/mitmproxy/mitmproxy/blob/240a286b2a3cedd0193b660e5217adfa46bb82b3/.github/python-version.txt
